app/uk/gov/hmrc/economiccrimelevyregistration/controllers/CheckYourAnswersController.scala

```diff
@@ -62,7 +62,9 @@ class CheckYourAnswersController @Inject() (
     ).flatten
   ).withCssClass("govuk-!-margin-bottom-9")
 
-  private def organisationDetails()(implicit request: RegistrationDataRequest[_]): SummaryList =
+  private def organisationDetails(liabilityRow: Option[SummaryListRow] = None)(implicit
```

Comment:
why are you passing it in as a parameter as opposed to just adding the row in the Seq?